webapp/portlet/src/main/webapp/dlp-quarantine/dlpAdministrationServices.js

```diff
@@ -25,3 +25,16 @@ export function isDlpFeatureActive() {
     }
   });
 }
+
+export function getDlpPositiveItems() {
+  return fetch(`${eXo.env.portal.context}/${eXo.env.portal.rest}/dlp/items`, {
```

Comment:
should we use offset and limit to prevent to get ALL items ?

Reply:
Updated !